src/agenda/index.js

```diff
@@ -16,7 +16,7 @@ import ReservationsList from './reservation-list';
 import styleConstructor from './style';
 import { VelocityTracker } from '../input';
 
-const HEADER_HEIGHT = 104;
+const HEADER_HEIGHT = 108;
```

Comment:
can't this be a prop instead?

Reply:
probably at lerast this should be different for different platform? If IOS is fine with 104

Reply:
Use like this

```jsx
import { Platform } from 'react-native'

const HEADER_HEIGHT = Platform.OS === 'ios' ? 104 : 108
```

Reply:
Use Platform api to get specific value

Reply:
The issue is still there for much smaller screens (tested with 3.4")
Created the following PR as a fix https://github.com/wix/react-native-calendars/pull/953